WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2272,6 +2276,11 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
 
         boolean isFreeWPCom = mSite.isWPCom() && SiteUtils.onFreePlan(mSite);
 
+        int editorOnboardingPhasePercentage = 0;
```

Comment:
Since this variable is a constant, traditional java practice would say that you should make this a static variable [at the top of the file](https://github.com/wordpress-mobile/WordPress-Android/blob/04e523e0505945507852b57db0f57d752bcc6b1a/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L297-L300) along these lines:

```
private static final int EDITOR_ONBOARDING_PHASE_PERCENTAGE = 0;
```

The reason for that is that it is slightly more memory efficient because it avoids the possibility of allocating the `int` multiple times. With that said, if you have reasons for not wanting to do that I certainly wouldn't consider this a blocker since this is an `Activity` class that isn't going to be instantiated a bunch of times.

Reply:
Ah, I was not aware of that ability or practice. I'll update the implementation to match your suggestion. Thank you!